compose/base.yml

```diff
@@ -94,10 +94,11 @@ services:
       - user_keys:/etc/sawtooth/keys
       - ${REMME_CONTAINER_EXPORTS_FOLDER:-./default_export}:/root/usr/share
       - ../config/remme-client-config.toml:/config/remme-client-config.toml
-      - ../config/remme-rest-api.toml:/config/remme-rest-api.toml
-      - ../config/log/remme-rest-api.toml:/etc/sawtooth/log_config.toml
-      - logs:/var/log
-    entrypoint: python3 -m remme.rest_api
+      - ../config/remme-rpc-api.toml:/config/remme-rpc-api.toml
+      - ../config/remme-log-config.toml:/etc/sawtooth/log_config.toml
```

Comment:
We have moved logging configuration to a separate directory `config/log` and removed the common config.

Reply:
@eugene-babichenko from dev branch
```
  remme-rest-api:
    container_name: remme_rest_api
    image: remme/remme-core:latest
    network_mode: "service:validator"
    environment:
      - REMME_CONTAINER_EXPORTS_FOLDER=./default_export
    volumes:
      - user_keys:/etc/sawtooth/keys
      - ${REMME_CONTAINER_EXPORTS_FOLDER:-./default_export}:/root/usr/share
      - ../config/remme-client-config.toml:/config/remme-client-config.toml
      - ../config/remme-rest-api.toml:/config/remme-rest-api.toml
      - ../config/log/remme-rest-api.toml:/etc/sawtooth/log_config.toml
      - logs:/var/log
    entrypoint: python3 -m remme.rest_api
```

Reply:
Oh I see